public/javascripts/EditProduct.js

```diff
@@ -0,0 +1,54 @@
+function showProductID(selectedID){
+    //TODO implement getQueryStringParams().inventoryID
+    // I have made selected id as 100 for now for testing purposes until i get the getQueryStringParams().inventoryID
+    //var selectedID = getQueryStringParams().inventoryID;
+   // var selectedID = "100";
+    console.log(selectedID);
+    var host = "http://localhost:50001/EditProduct/" +selectedID +"/";
+
+    sendRequest(host, function() {
+        if(dispReq.readyState == 4 && dispReq.status == 200) {
+            var selectedProduct = jQuery.parseJSON(dispReq.responseText);
```

Comment:
Please put a todo in here to leverage John's new breadcrumb subpage loader to load a new page populated with the data here.